src/bindings/c/ov/src/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME openvino_ov_c)
```

Comment:
Why two openvino words? 😄 
Can we use openvino_c?

Reply:
It is used to distinguish legacy C API, we will name it openvino_c after move legacy C API to legacy-like directory.